configs/face_2d_keypoint/topdown_regression/wflw/td-reg_res50_8x64e-210e_wflw-256x256.py

```diff
@@ -0,0 +1,121 @@
+_base_ = [
+    '../../../_base_/default_runtime.py', '../../../_base_/datasets/wflw.py'
+]
+
+# runtime
+train_cfg = dict(max_epochs=210, val_interval=1)
+
+# optimizer
+optim_wrapper = dict(optimizer=dict(
+    type='Adam',
+    lr=5e-4,
+))
+
+# learning policy
+param_scheduler = [
+    dict(
+        type='LinearLR', begin=0, end=500, start_factor=0.001,
+        by_epoch=False),  # warm-up
+    dict(
+        type='MultiStepLR',
+        begin=0,
+        end=210,
+        milestones=[170, 200],
+        gamma=0.1,
+        by_epoch=True)
+]
+
+# automatically scaling LR based on the actual training batch size
+auto_scale_lr = dict(base_batch_size=512)
+
+# codec settings
+codec = dict(type='RegressionLabel', input_size=(256, 256))
+
+# model settings
+model = dict(
+    type='TopdownPoseEstimator',
+    data_preprocessor=dict(
+        type='PoseDataPreprocessor',
+        mean=[123.675, 116.28, 103.53],
+        std=[58.395, 57.12, 57.375],
+        bgr_to_rgb=True),
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50'),
+    ),
+    neck=dict(type='GlobalAveragePooling'),
+    head=dict(
+        type='RegressionHead',
+        in_channels=2048,
+        num_joints=98,
+        loss=dict(type='SmoothL1Loss', use_target_weight=True),
+        decoder=codec),
+    train_cfg=dict(),
+    test_cfg=dict(
+        flip_test=True,
+        shift_coords=True,
+    ))
+
+# base dataset settings
+dataset_type = 'WFLWDataset'
+data_mode = 'topdown'
+data_root = 'data/wflw/'
+
+# pipelines
+train_pipeline = [
+    dict(type='LoadImage'),
+    dict(type='GetBBoxCenterScale'),
+    dict(type='RandomFlip', direction='horizontal'),
+    dict(type='RandomBBoxTransform', scale_factor=[0.25], rotate_factor=80),
```

Comment:
`scale_factor` should be a `Tuple[float, float]`

Reply:
> `scale_factor` should be a `Tuple[float, float]`

Got it. Also, I noticed that the rotate_factor here was incorrect. I changed this part to:
```python
# 1.x
dict(
        type='RandomBBoxTransform',
        scale_factor=[0.75, 1.25],
        rotate_factor=60),

# 0.x
dict(
        type='TopDownGetRandomScaleRotation', rot_factor=30,
        scale_factor=0.25),
```